sdk/src/Core/Amazon.Runtime/CredentialManagement/CredentialProfileOptions.cs

```diff
@@ -121,6 +121,11 @@ public class CredentialProfileOptions
         /// </summary>
         public string SsoRegion { get; set; }
 
+        /// <summary>
+        /// The SSO scopes that are provided for authorization when using AWS SSO.
+        /// </summary>
+        public string SsoRegistrationScopes { get; set; }
```

Comment:
Can you update the `ToString`, `Equals` and `HashCode` methods to take include this new property.

Reply:
Done

Reply:
And can you update `IsEmpty` as well?

Reply:
Done